frontend/app/controllers/Healthcheck.scala

```diff
@@ -1,34 +1,58 @@
 package controllers
 
-import com.github.nscala_time.time.Imports._
+import com.gu.membership.model.{FriendTierPlan, PaidTierPlan, StaffPlan}
+import com.gu.membership.salesforce.Tier.{Partner, Patron, Supporter}
 import com.gu.monitoring.CloudWatchHealth
 import play.api.Logger
+import play.api.libs.concurrent.Execution.Implicits._
 import play.api.mvc.{Action, Controller}
 import services.{GuardianLiveEventService, TouchpointBackend}
 
-case class Test(name: String, ok: () => Boolean)
+import scala.concurrent.Await
+import scala.util.{Try, Failure, Success}
+
+case class Test(name: String, exec: () => Boolean) {
+  def failedTest() = Logger.warn(s"Test $name failed, health check will fail")
+
+  def ok() = {
+    val passed = exec()
+    if (!passed) Logger.warn(s"Test $name failed, health check will fail")
+    passed
+  }
+}
+
+object ZuoraPing {
+  import com.github.nscala_time.time.Imports._
+  val zuoraSoapService = TouchpointBackend.Normal.zuoraSoapService
+  def ping = zuoraSoapService.lastPingTimeWithin(2.minutes)
+}
 
 object Healthcheck extends Controller {
-  val zuoraService = TouchpointBackend.Normal.zuoraSoapService
+  import scala.concurrent.duration._
+
+  val zuoraSoapService = TouchpointBackend.Normal.zuoraSoapService
+  val subscriptionService = TouchpointBackend.Normal.subscriptionService
 
-  val tests = Seq(
+  val productRatePlanTiers = List(FriendTierPlan, StaffPlan,
```

Comment:
Since we have more tiers that just theses 6, I think that we'll be adding to many operations that abstract away and that explicitly specifying them in a single list is more straightforward.